Package.swift

```diff
@@ -29,8 +29,7 @@ let package = Package(
         ),
         .testTarget(
             name: "CacheAdvanceTests",
-            dependencies: ["CacheAdvance", "LorumIpsum"],
-            swiftSettings: [.define("SWIFT_PACKAGE_MANAGER")]
```

Comment:
Interesting. Is the reason that we don't need this because we don't distribute the tests via CocoaPods?

Reply:
Correct!